diff --git a/node_modules/react-native-wheel-of-fortune/assets/images/knob.png b/node_modules/react-native-wheel-of-fortune/assets/images/knob.png
new file mode 100644
index 0000000..78832cc
Binary files /dev/null and b/node_modules/react-native-wheel-of-fortune/assets/images/knob.png differ
diff --git a/node_modules/react-native-wheel-of-fortune/src/index.js b/node_modules/react-native-wheel-of-fortune/src/index.js
index 0cdb2a1..ec150fd 100644
--- a/node_modules/react-native-wheel-of-fortune/src/index.js
+++ b/node_modules/react-native-wheel-of-fortune/src/index.js
@@ -36,6 +36,7 @@ class WheelOfFortune extends Component {
   prepareWheel = () => {
     this.Rewards = this.props.options.rewards;
     this.RewardCount = this.Rewards.length;
+    this.getWinner = this.props.options.getWinner;
 
     this.numberOfSegments = this.RewardCount;
     this.fontSize = 20;
@@ -138,9 +139,15 @@ class WheelOfFortune extends Component {
     );
   };
 
+//   getWinner = (value, index) => {
+//     this.setState({
+//         winnerValue: value,
+//         winnerIndex: index,
+//     });
+//   }
+
   _onPress = () => {
     const duration = this.props.options.duration || 10000;
-
     this.setState({
       started: true,
     });
@@ -157,7 +164,8 @@ class WheelOfFortune extends Component {
         finished: true,
         winner: this._wheelPaths[winnerIndex].value,
       });
-      this.props.getWinner(this._wheelPaths[winnerIndex].value, winnerIndex);
+      console.log(this.getWinner)
+      this.getWinner(this._wheelPaths[winnerIndex].value, winnerIndex);
     });
   };
 
@@ -312,8 +320,6 @@ class WheelOfFortune extends Component {
           <Image
             source={
               this.props.options.knobSource
-                ? this.props.options.knobSource
-                : require('../assets/images/knob.png')
             }
             style={{ width: knobSize, height: (knobSize * 100) / 57 }}
           />
